Package.swift

```diff
@@ -47,13 +47,13 @@ if ProcessInfo.processInfo.environment["YSWIFT_LOCAL"] != nil {
             name: "yniffiFFI",
             path: "./lib/yniffiFFI.xcframework"
     )
-} else {
-    FFIbinaryTarget = .binaryTarget(
-            name: "yniffiFFI",
-            url: "https://github.com/y-crdt/yswift/releases/download/0.2.0/yniffiFFI.xcframework.zip",
-            checksum: "d2633bdb1e9f257cd56a852f360f0d0f4bc1615a4c34a05e76a2da2c430a0f98"
-    )
-}
+//} else {
```

Comment:
Left this like this intentionally to not forget to update the xcframework before merging

Reply:
You should be able to go ahead and drop this back into place, as the CI system builds with the local build all the time, to allow us to test and validate changes - and we'll want likely want to separate all these changes from the process of cutting a new release, which updates the checksums, tags, etc for those various bits.